traffic_portal/app/src/modules/private/serviceCategories/index.js

```diff
@@ -21,7 +21,7 @@ module.exports = angular.module('trafficPortal.private.serviceCategories', [])
     .config(function($stateProvider, $urlRouterProvider) {
         $stateProvider
             .state('trafficPortal.private.serviceCategories', {
-                url: 'servicecategories',
+                url: 'service_categories',
```

Comment:
sorry, should be `service-categories` if you want to be consistent with the other TP urls. also, i guess it's a agreed upon web convention:

https://www.ecreativeim.com/blog/index.php/2011/03/30/seo-basics-hyphen-or-underscore-for-seo-urls/#:~:text=The%20short%20answer%20is%20that,same%20as%20redsneakers%20to%20Google.&text=Again%2C%20SEO%20URLs%20should%20use%20hyphens%20to%20separate%20words.